src/main/java/it/gov/pagopa/pu/pagopapayments/config/SoapWebServiceConfig.java

```diff
@@ -0,0 +1,193 @@
+package it.gov.pagopa.pu.pagopapayments.config;
+
+import it.gov.pagopa.pu.pagopapayments.exception.ApplicationException;
+import it.gov.pagopa.pu.pagopapayments.ws.server.PaForNodeEndpoint;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.web.servlet.ServletRegistrationBean;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.ws.config.annotation.EnableWs;
+import org.springframework.ws.config.annotation.WsConfigurerAdapter;
+import org.springframework.ws.support.WebUtils;
+import org.springframework.ws.transport.http.MessageDispatcherServlet;
+import org.springframework.ws.transport.http.WsdlDefinitionHandlerAdapter;
+import org.springframework.ws.wsdl.WsdlDefinition;
+import org.springframework.ws.wsdl.wsdl11.SimpleWsdl11Definition;
+import org.springframework.ws.wsdl.wsdl11.Wsdl11Definition;
+import org.springframework.xml.xsd.SimpleXsdSchema;
+import org.springframework.xml.xsd.XsdSchema;
+import org.springframework.xml.xsd.XsdSchemaCollection;
+
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+@EnableWs
+@Slf4j
+@Configuration(proxyBeanMethods = false)
+public class SoapWebServiceConfig extends WsConfigurerAdapter {
+
+  @Value("${app.absolute-base-url}")
+  private String appAbsolutePath;
```

Comment:
Can we give a more meaningful name? Is it the baseUrl exposed to the pagoPa node?
```suggestion
  @Value("${app.pagopa-payments-ws-base-url}")
  private String pagopaPaymentsWsBaseUrl;
```

Reply:
done